source/_docs/composer.md

```diff
@@ -1,175 +1,87 @@
+
+In this workflow, a [Multidev](/docs/multidev/) environment is created on Pantheon for each pull request branch on GitHub. Work in these environments can also be committed back to the same branch for review on GitHub. When a pull request is merged into the default branch on GitHub, the result is deployed to the Dev environment on Pantheon:
+
+![Multidev PR workflow](/source/docs/assets/images/pr-workflow/github-pr-diagram.png)
+
+### Automated Tests
+
+It is also common to set up automated tests to confirm that the project is working as expected; when tests are available, GitHub will run them and display the results of the tests with the pull request. Working on projects with comprehensive tests increases the development team's confidence that submitted pull requests will work correctly when they are integrated into the main build.
+
+### Scaling Considerations
+We recommend the "source" and "artifact" workflow for single site use cases, and for most use cases involving larger site portfolios such as EDUs. You can create a scaffold repository based off our example repositories and customize it to your liking, then use the scaffold to create new sites similar to how you would create the codebase for a [Custom Upstream](/docs/custom-upstream/).
+
+However, unlike Custom Upstreams this method does not support one-click updates in the Site Dashboard. Adopting this workflow means forgoing all other update techniques in favor of Composer. If your use case requires an simpler update strategy for non-technical site admins, this workflow could present problems scaling or at the very least require additional training for your development team.
+
+## Custom Upstream Workflow
+It is possible to preserve the functionality of Pantheon's one-click updates in the Site Dashboard for Composer managed sites, however it's use case is quite narrow. Your scaffold repository based off our example repositories would need to commit all dependencies to the Custom Upstream repository. Updates via Composer would only happen at the Custom Upstream repository level, which would then trickle down to sites created from the Custom Upstream as one-click updates.
+
+This workflow has one very serious shortcoming, that is site-specific dependencies are likely to cause a lot of conflicts. The practical use case for this workflow is for a large group of sites that require a single set of dependencies. You should only use this method if you don’t intend to use site specific themes, modules, or plugins downstream.
+
+## Next Steps
```

Comment:
Should we add a CTA for the enablement page? Seems like composer based workflows at scale is something we'd want to offer training for, especially since use cases vary